library/src/main/java/com/opensource/svgaplayer/drawer/SVGACanvasDrawer.kt

```diff
@@ -369,6 +365,24 @@ internal class SVGACanvasDrawer(videoItem: SVGAVideoEntity, val dynamicItem: SVG
             return sharedMatrix2
         }
 
+        fun shareMattePaint(): Paint {
+            shareMattePaint.setXfermode(PorterDuffXfermode(PorterDuff.Mode.DST_IN))
+            return shareMattePaint
+        }
+
+        fun sharedMatteBitmap(): Bitmap {
+            return sharedMatteBitmap as Bitmap
+        }
+
+        fun shareMatteCanvas(width: Int, height: Int): Canvas {
+            if (shareMatteCanvas == null) {
+                sharedMatteBitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ALPHA_8)
```

Comment:
Bitmap not always with same width and height, and bitmap content may change if matte content diff.

Reply:
Did avoid null matte bitmap.